taskt/Core/Automation/Commands/Outlook Commands/GetOutlookEmailsCommand.cs

```diff
@@ -1,109 +1,115 @@
-﻿using System;
-using System.Linq;
-using System.Xml.Serialization;
-using System.Data;
-using System.Windows.Forms;
+﻿using Microsoft.Office.Interop.Outlook;
+using System;
 using System.Collections.Generic;
-using taskt.UI.Forms;
+using System.IO;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
 using taskt.UI.CustomControls;
-using Microsoft.Office.Interop.Outlook;
+using taskt.UI.Forms;
 using Application = Microsoft.Office.Interop.Outlook.Application;
-using taskt.Core.Utilities.CommonUtilities;
 
 namespace taskt.Core.Automation.Commands
 {
     [Serializable]
-    [Attributes.ClassAttributes.Group("Outlook Commands")]
-    [Attributes.ClassAttributes.Description("This command allows you to get emails and attachments with outlook")]
-    [Attributes.ClassAttributes.UsesDescription("Use this command when you want to get emails and attachments with your currenty logged in outlook account")]
-    [Attributes.ClassAttributes.ImplementationDescription("")]
+    [Group("Outlook Commands")]
+    [Description("This command gets selected emails and their attachments in Outlook.")]
```

Comment:
change to "This command gets selected emails and their attachments **from** Outlook."

Reply:
done